docs/_includes/common/filter-menu.html

```diff
@@ -0,0 +1,472 @@
+<div id="filter-menu">
+    <h2><span class="secondary-text">@ebay/skin/</span>filter-menu</h2>
+
+    <h3 id="filter-menu-default">Default Filter Menu</h3>
+    <p>The default filter menu is a <em>stateless</em> menu.</p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <span class="filter-menu">
+                <div class="filter-menu__items" role="menu">
+                    <div class="filter-menu__scroll-area">
+                        <div class="filter-menu__item" role="menuitem">
```

Comment:
For the ARIA version, this should be `menuitemcheckbox` instead of `menuitem`.

Reply:
Correct. I will make another example with the form version.